### PR TITLE
Replace Gitter link with Slack channel link

### PR DESCRIPTION
**Pull request by akeel230**
## Purpose
fix issue #1005 

## Goals
Help contributors lead towards active community channel

##  Checklist
- [X] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [X] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by anjula-sack** [comment]
Can you update the title? `Replace Gitter link with Slack channel link`

**Comment by akeel230** [comment]
@anjula-sack tittle  changed

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ channels (wikipedia, stackoverflow) but also do not hesitate to ask questions to
 
 ## Communication Channels
 
-Currently, we use Gitter as our communication channel. [Join SEF Gitter Chanel](https://gitter.im/sef-global)
+Currently, we use Slack as our communication channel. [Join SEF Slack Channel](http://sefheadquarters.slack.com)
```
Comment (anjula-sack) [inline CONTRIBUTING.md]: http -> https

File: README.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ This repository contains source for Sustainable Education Foundation Website. [V
 
 * Discourse: https://sef.discourse.group/
 * Leaderboard: https://sefglobal.org/github-leaderboard/
+* Slack channel: http://sefheadquarters.slack.com
```
Comment (anjula-sack) [inline README.md]: https

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (akeel230) [reply CONTRIBUTING.md]: done

File: README.md (hunk 2 above)
Comment (akeel230) [reply README.md]: changes made

**Comment by akeel230** [comment]
now?
@anjula-sack

**Pull request closed without merging**